previews/primer/beta/button_preview.rb

```diff
@@ -20,7 +20,6 @@ class ButtonPreview < ViewComponent::Preview
       # @param disabled toggle
       # @param inactive toggle
       # @param align_content select [center, start]
-      # @param tag select [a, summary, button]
```

Comment:
Since tag is still an option for the component, I think we should at least keep it in the Playground story. This is what we use to test different cases and make sure we haven't broken anything.. I would at least keep it to `button, a` and maybe there's a way to pass in `href` if `a` is selected?

Reply:
I can definitely keep it to `button` and `a`! There is a way I can dynamically add a `#` if it is an `<a>` so I will do that :) Thank you!